### PR TITLE
Feature/auth component

### PR DESCRIPTION
**Pull request by DoctorFreed**

### CODE REVIEW

File: src/app/authorization/authorization.component.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.auth-comp {
+  width: 30vw;
+  height: 20vh;
+
```
Comment (usatovaDasha) [inline src/app/authorization/authorization.component.scss:4]: why need this empty line?

File: src/app/authorization/authorization.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          this.errAuth = false;
+        } else {
+          this.errAuth = true;
+        }
+      });
+  }
+
+  goReg(): void {}
+
+  ngOnInit(): void {}
```
Comment (usatovaDasha) [inline src/app/authorization/authorization.component.ts:42]: why need empty functions?

File: src/app/person.service.ts [hunk 3]
```diff
@@ -18,6 +18,11 @@ export class PersonService {
   logining(login: string, password: string): Observable<any> {
     return this.http
       .post(`http://localhost:8000/person/login`, { login, password })
-      .pipe(map((result: any) => result));
+      .pipe(
+        map((result: any) => result),
+        catchError((err: any) => {
+          return [err.message];
+        })
```
Comment (usatovaDasha) [inline src/app/person.service.ts:25]: ;

File: src/app/authorization/authorization.component.ts (hunk 2 above)
Comment (DoctorFreed) [reply src/app/authorization/authorization.component.ts:42]: goReg redireted to reg page, but route not eat ready, i add alert

File: src/app/authorization/authorization.component.scss (hunk 1 above)
Comment (DoctorFreed) [reply src/app/authorization/authorization.component.scss:4]: fixed

**Pull request merged**